lib/logstash/filters/xml.rb

```diff
@@ -35,29 +35,30 @@ class LogStash::Filters::Xml < LogStash::Filters::Base
   #
   # XML in the value of the source field will be expanded into a
   # datastructure in the `target` field.
-  # Note: if the `target` field already exists, it will be overridden
-  # Required
+  # Note: if the `target` field already exists, it will be overridden.
+  # Required.
   config :target, :validate => :string
```

Comment:
Okay. Since you filed a separate issue I wasn't sure whether you'd take care of that change when addressing that issue. I'll update the commit shortly.